internal/api/cron/wallet.go

```diff
@@ -235,80 +238,60 @@ func (h *WalletCronHandler) CheckAlerts(c *gin.Context) {
 					"any_balance_below", isAnyBalanceBelowThreshold,
 				)
 
-				// Handle balance above threshold (recovery)
-				if !isAnyBalanceBelowThreshold {
-					h.logger.Infow("all balances above threshold - checking recovery",
-						"wallet_id", wallet.ID,
-						"threshold", threshold,
-						"current_balance", currentBalance,
-						"ongoing_balance", ongoingBalance,
-						"alert_state", wallet.AlertState,
-					)
-
-					// If current state is alert, update to ok (recovery)
-					if wallet.AlertState == string(types.AlertStateAlert) {
-						if err := h.walletService.UpdateWalletAlertState(ctx, wallet.ID, types.AlertStateOk); err != nil {
-							h.logger.Errorw("failed to update wallet alert state",
-								"wallet_id", wallet.ID,
-								"error", err,
-							)
-						} else {
-							h.logger.Infow("wallet recovered from alert state",
-								"wallet_id", wallet.ID,
-							)
-						}
-					}
-					continue
-				}
-
-				// Skip if already in alert state
-				if wallet.AlertState == string(types.AlertStateAlert) {
-					h.logger.Infow("skipping wallet - already in alert state",
-						"wallet_id", wallet.ID,
-					)
-					continue
+				// Determine alert status based on balance check
+				var alertStatus types.AlertState
+				if isAnyBalanceBelowThreshold {
+					alertStatus = types.AlertStateInAlarm
+				} else {
+					alertStatus = types.AlertStateOk
 				}
 
-				h.logger.Infow("balance below/equal threshold - triggering alert",
+				h.logger.Infow("logging alert status",
 					"wallet_id", wallet.ID,
 					"threshold", threshold,
 					"current_balance", currentBalance,
 					"ongoing_balance", ongoingBalance,
+					"alert_status", alertStatus,
+					"current_alert_state", wallet.AlertState,
 				)
 
-				// Update wallet state to alert
-				if err := h.walletService.UpdateWalletAlertState(ctx, wallet.ID, types.AlertStateAlert); err != nil {
```

Comment:
```
map[alert_type][alert_status] = {
    "webhook_event" : "event name"
}

map[low_bal][ok] = {
    "webhook_event" : "walllet.balance.recovered"
}

map[low_bal][in_alarm] = {
    "webhook_event" : "walllet.balance.dropped"
}
```

Need to fire the old webhooks only not the new alert ones
also ensure we fire webhook only when difference in desired vs current status